# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/tests/crashtests/1585851.html
# Commit: 61e813c94bc9
# Full Hash: 61e813c94bc98cd2ebb03113fb3c1c1bf02283c8
# Author: James Teh <jteh@mozilla.com>
# Date: 2019-10-30 15:06:46
# Description:
#   Bug 1585851: When processing a11y insertions, don't try to move an accessible if its new previous sibling hasn't been moved into its new container yet. r=MarcoZ
#   
#   Sometimes, depending on how children were changed, children might be in the insertion list out of order; e.g. [child2, child1].
#   It's also possible that an earlier child (child1 in the above example) is being moved out of another container.
#   When processing the earlier insertion (child2), we'll determine we need to move it within its parent and will fetch its new previous sibling so we can move it into the right place.
# ==============================================================================

diff -r 1ee997d2152a -r 61e813c94bc9 accessible/tests/crashtests/1585851.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/accessible/tests/crashtests/1585851.html	Wed Oct 30 06:32:50 2019 +0000
@@ -0,0 +1,21 @@
+<style>
+* {
+  border-collapse: collapse;
+}
+</style>
+<script>
+window.onload = () => {
+  c.style.setProperty("border-collapse", "separate")
+}
+function eh() {
+  a.options.add(f)
+  a.add(b, 9)
+  a.add(f, d)
+}
+</script>
+<select id="a"></select>
+<option id="b">
+<dd id="c">
+<audio id="d" controls>
+<details open="true" ontoggle="eh()">
+<optgroup id="f">